app/controllers/idv/hybrid_mobile/document_capture_controller.rb

```diff
@@ -82,6 +85,10 @@ def redo_document_capture_pending?
 
         document_capture_session.requested_at > stored_result.captured_at
       end
+
+      def phone_with_camera
+        { phone_with_camera: phone_question_ab_test_bucket == :show_phone_question ? true : nil }
+      end
```

Comment:
1. I think this should move into `phone_question_ab_test_concern`
2. I think that everything we do here in `hybrid_mobile/document_capture_controller.rb` we also need to do in `idv/document_capture_controller.rb` ?

Reply:
@charleyf - this is present because we don't have idv session .. this logic is exclusive to the hybrid document capture